packages/amplify-category-notifications/src/pinpoint-helper.ts

```diff
@@ -547,7 +547,7 @@ export const getPinpointClient = async (
   if (httpProxy) {
     aws.config.update({
       httpOptions: {
-        agent: proxyAgent(httpProxy),
+        agent: new ProxyAgent(),
```

Comment:
is it ok to drop `httpProxy` param here ?

it seems to be a pattern in the PR.

Reply:
yeah see description for more details; tldr it reads from process.env by default so we don't need to manually specify anymore

Reply:
if we read these variables anyway for the `if` I'd rather pass this explicitly.
unless they took away API to do this. In that case might be worth to add comment for future reader why `httpProxy` is not consumed.

Reply:
added notes